lib/Parse/ParseDecl.cpp

```diff
@@ -3262,6 +3262,48 @@ void Parser::parseDeclDelayed() {
   });
 }
 
+ParserStatus Parser::parseAccessPath(
+    SmallVectorImpl<ImportDecl::AccessPathElement> &AccessPath,
+    const Diagnostic &D) {
+
+  bool HasNext;
+  do {
+    SyntaxParsingContext AccessCompCtx(SyntaxContext,
+                                       SyntaxKind::AccessPathComponent);
+    if (Tok.is(tok::code_complete)) {
+      consumeToken();
+      if (CodeCompletion) {
+        CodeCompletion->completeImportDecl(AccessPath);
+      }
+      return makeParserCodeCompletionStatus();
+    }
+    AccessPath.push_back(std::make_pair(Identifier(), Tok.getLoc()));
+    SourceLoc L;
```

Comment:
Nitpick: 'Loc' might be a better name than 'L', it's more common for variables of SourceLoc type to be named Loc than L.